types/react-native/index.d.ts

```diff
@@ -740,7 +740,7 @@ export interface LayoutChangeEvent {
     }
 }
 
-export interface TextStyleIOS extends ViewStyle {
+export interface TextStyleIOS extends ViewStyle  {
```

Comment:
FIxed,  sorry, as https://github.com/DefinitelyTyped/DefinitelyTyped/pull/17621#discussion_r124844762 said, It's just a mistake.